ArduPlane/quadplane.cpp

```diff
@@ -556,6 +556,22 @@ const AP_Param::GroupInfo QuadPlane::var_info2[] = {
     // @User: Standard
     AP_GROUPINFO("APPROACH_DIST", 39, QuadPlane, approach_distance_m, 0),
     
+#if HAL_WITH_ESC_TELEM
+    // @Param: TKOFF_RPM_MIN
+    // @DisplayName: Takeoff Check RPM minimum
+    // @Description: Takeoff is not permitted until motors report at least this RPM.  Set to zero to disable check
```

Comment:
I think we need to make this clear its auto takeoff only. Copter is all takeoffs I think.

Reply:
Ah, stale description from the original implementation. Good catch